### PR TITLE
Finished testing & last story regarding restrictions in the search page + other css stuff

### PR DESCRIPTION
**Pull request by cmorris19**

### CODE REVIEW

File: MealFridge/Models/Partials/RecipePartial.cs [hunk 1]
```diff
@@ -1,12 +1,19 @@
...
 using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations.Schema;
 using System.Linq;
 using System.Threading.Tasks;
 
 namespace MealFridge.Models
 {
     public partial class Recipe : IFoodItem
     {
+
```
Comment (Jutter18) [inline MealFridge/Models/Partials/RecipePartial.cs:12]: Why all of this?

File: MealFridge/Models/Repositories/SpnApiService.cs [hunk 2]
```diff
@@ -158,25 +158,27 @@ private static Recipe GetDetailRecipe(JObject recipeDetails)
 
         private string SendRequest()
         {
-            try
+            var request = (HttpWebRequest)WebRequest.Create(_query.GetUrl);
```
Comment (Jutter18) [inline MealFridge/Models/Repositories/SpnApiService.cs:161]: Why this change as well

File: MealFridge/Views/Home/Index.cshtml [hunk 3]
```diff
@@ -43,7 +41,7 @@
                                     <div class="card">
                                         <img src="@Model[i].Image" alt="Recipe image" class="card-img-top" />
                                         <h5 class="card-title">@Model[i].Title</h5>
-                                        <p class="card-text">@Model[i].Summery</p>
```
Comment (Jutter18) [inline MealFridge/Views/Home/Index.cshtml:46]: Why'd we remove this?

File: MealFridge/Views/Search/RecipeCards.cshtml [hunk 4]
```diff
@@ -9,45 +9,53 @@
...
-                         <button type="button"
-                                 id="favButton"
-                                 class="btn btn-outline-success"
-                                 style="display:inline-block"
-                                 onclick="addFavorite('@r.Id')" title="Favorite recipe">
-                             <i class="fa fa-heart"> @r.Savedrecipes.Where(f => f.RecipeId == r.Id && f.Favorited != false).Count()</i>
-                         </button>
+<div class="card shadow-lg">
+    <img class="card-img-top" src="@r.Image" alt="Recipe Image">
+    @* THERE IS A BUG DO NOT TOUCH UNLESS YOU ARE SMARTER THAN CHRISTIAN @if (temp.Favorited)
```
Comment (Jutter18) [inline MealFridge/Views/Search/RecipeCards.cshtml:18]: Gonna need more explanation on this one

File: MealFridge/Models/Partials/RecipePartial.cs (hunk 1 above)
Comment (cmorris19) [reply MealFridge/Models/Partials/RecipePartial.cs:12]: Added partial so that I could flag a bool which helped me complete my third story with restrictions being accounted for in the search recipes function.

File: MealFridge/Models/Repositories/SpnApiService.cs (hunk 2 above)
Comment (cmorris19) [reply MealFridge/Models/Repositories/SpnApiService.cs:161]: This was just a resulted code change from internal bug fixing. Will make to pull request to update it.

File: MealFridge/Views/Home/Index.cshtml (hunk 3 above)
Comment (cmorris19) [reply MealFridge/Views/Home/Index.cshtml:46]: Wasn't showing up anyways, but just like the try-catch code bit above, I'll put it back in a new pull request.

File: MealFridge/Views/Search/RecipeCards.cshtml (hunk 4 above)
Comment (cmorris19) [reply MealFridge/Views/Search/RecipeCards.cshtml:18]: Made a note that there's been a bug from a previous sprint in this logic that every saved recipe will occasionally flag all search recipes with a heart. Commented it out for this sprint in order to show that my restrictions being taken into account works.
